src/options.ts

```diff
@@ -48,6 +48,10 @@ export interface GtmSupportOptions {
    * @default false
    */
   compatibility?: boolean;
+  /**
+   * Will add specified MIME type to script tag.
+   */
+  scriptType?: string;
```

Comment:
Are there specific scriptTypes that can be used as string literal union here?

Reply:
I'm not really sure.

I think there are some standard values ([here](https://www.w3schools.com/jsref/prop_script_type.asp), [here](https://www.iana.org/assignments/media-types/media-types.xhtml)). But the list is not restrictive, in a sense that any other values are also allowed.

For example, the type that I need is `text/partytown` and it's not among the standard ones.

So I guess `string` typescript type should be enough here.